pages/linux/procs.md

```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
```

Comment:
```suggestion
- List all processes showing the: PID, user, CPU usage, memory usage and, the command which started them:
```

Reply:
I don't agree with the `:` and the comma should go before the `and`:
```suggestion
- List all processes showing the PID, user, CPU usage, memory usage, and the command which started them:
```